apps/rehabstod/src/components/error/GetTableContentError/GetTableContentError.tsx

```diff
@@ -0,0 +1,22 @@
+import { FetchBaseQueryError } from '@reduxjs/toolkit/query'
+import { SerializedError } from '@reduxjs/toolkit'
+import { ErrorAlert } from '../ErrorAlert/ErrorAlert'
+import { getTableErrorDescription, getTableErrorTitle } from '../../Table/utils/tableTextGeneratorUtils'
+
+export function GetTableContentError({
+  tableName,
+  error,
+}: {
+  tableName: string
+  error: (FetchBaseQueryError & { id?: string }) | (SerializedError & { id?: string })
+}) {
+  return (
+    <ErrorAlert
+      heading={getTableErrorTitle(tableName)}
```

Comment:
This returns a text that is not reused anywhere else. Why not include it in this component?